File: android-lib-dagger2/dagger2-sample/app/src/main/java/com/chiclaim/dagger/sample/MyApplication.java
package com.chiclaim.dagger.sample;

import android.app.Application;

/**
 * Description：
 * <br/>
 * Created by kumu on 2017/2/27.
 */

public class MyApplication extends Application {


    @Override
    public void onCreate() {
        super.onCreate();
        ComponentManager.getInstance().init(this);
    }

}

File: android-archietecture/jetpack/jetpack-sample/app/src/main/java/com/chiclaim/jetpack/MyApplication.kt
package com.chiclaim.jetpack

import android.app.Application
import android.content.Context
import android.util.Log

class MyApplication : Application() {

    override fun attachBaseContext(base: Context?) {
        super.attachBaseContext(base)
        Log.e("MyApplication", "MyApplication attachBaseContext()")

    }

    override fun onCreate() {
        super.onCreate()
        Log.e("MyApplication", "MyApplication onCreate()")
    }


}